test/utils/config.go

```diff
@@ -187,3 +192,36 @@ func testChangePort_mixedGlobalApp(t *testing.T, snap, app, servicePort string)
 	})
 
 }
+
+func TestAutoStart(t *testing.T, snapName string, testAutoStart bool) {
+	t.Run("autostart", func(t *testing.T) {
+		if testAutoStart {
+			TestProcessAutostartGlobal(t, snapName)
+		}
+	})
+}
+
+func TestProcessAutostartGlobal(t *testing.T, snapName string) {
+	t.Run("set and unset autostart", func(t *testing.T) {
+		t.Cleanup(func() {
+			SnapUnset(t, snapName, "autostart")
+			SnapStop(t, snapName)
+		})
+
+		SnapStop(t, snapName)
+		require.False(t, SnapServicesEnabled(t, snapName))
+		require.False(t, SnapServicesActive(t, snapName))
```

Comment:
This is testing the snap stop. Not really necessary here, but feel free to keep.

Reply:
It is not aimed to test the snap stop. It makes sure the service inactive/disabled, to prepare for the following autostart test.